### ISSUE TITLE
Using AzCopy improved shows a failure icon

### ISSUE THREAD

**Issue by DonFrazier** [microsoft/AzureStorageExplorer#1490]
**Storage Explorer Version**: 1.8.1
**Build Number**: 20190510.6
**Platform/OS**: Windows 10
**Architecture**: ia32
**Regression From**:  

## Bug Description ##
Copying a large folder structure from Blob to local machine is great!  I'm using the new AzCopy and it's super fast.  Thanks.

When the copy finishes I have the red icon with a white X.  Message says Transfrom  from "source" to "destinatin" complete with skips: 0 Items transferred, xx,xxx items skipped (used SAS, discovery completed)

This implies a failure when it seems like there was really just nothing to copy.  The azcopy command generated was something like $env:AZCOPY_CRED_TYPE = "Anonymous";
./azcopy.exe copy "https://x.blob.core.windows.net/y/?se=201D" "A:\b" --overwrite=false --recursive --from-to=BlobLocal --list-of-files "C:\Users\user\AppData\Local\Temp\stg-exp-azcopy-.json" --check-md5 "FailIfDifferent";
$env:AZCOPY_CRED_TYPE = "";

## Steps to Reproduce ##
1. Select a blob with > 40 containers and >10K files
2. Download all
## Expected Experience ##
Expected a success
## Actual Experience ##
The red x icon suggests a failure when in reality this was the 2nd attempt and no new files had been created in the account.  Copying no files should not be a failure.

**Comment by craxal** [comment]
If transfer only has skipped items, we should use the same icon as we use for JS download skips.

**Comment by DonFrazier** [comment]
I would say if the issue is only that `all or some` items were skipped should the red icon appear.  I am refreshing my local copy of the 10K files and expect that 99% will be skipped every day.  That shouldn't be a failure.  But I also got the AZCOPY from the command line and it works just a well with more details about what's going on.

The red failure icon should be reserved for connection failures, actual download attempts that failed after retry was exhausted, etc.

**Comment by MRayermannMSFT** [comment]
Hey @DonFrazier , I'd like to get your opinion on this since you opened the bug. With the JS upload, we have children activities. So on the parent one we use the green check mark, and on the child (the skip) we use info:
![image](https://user-images.githubusercontent.com/21112697/60611555-6dc9f100-9d7b-11e9-8468-c1eb99edcace.png)

For AzCopy, we don't have children activities (and we aren't going to add any, at least not today). So for the AzCopy activity, would you want green check mark or info? What I'm thinking is:

- If we're only transferring one thing, then use info
- If we're transferring multiple things, use green check mark

What do you think?

**Comment by DonFrazier** [comment]
The command line output looks like this:

```
Job ... summary
Elapsed Time (Minutes): 0.1676
Total Number Of Transfers: 11741
Number of Transfers Completed: 37
Number of Transfers Failed: 0
Number of Transfers Skipped: 11704
TotalBytesTransferred: 285517818
Final Job Status: CompletedWithSkipped
```

If the UI has similar info in its response then this is a green check mark success state IMO.  It would be nice to see the numbers in a 1-line summary though.  Xferred 37 skipped 11,704 of 11,741 in 9.2 seconds.

**Comment by MRayermannMSFT** [comment]
Yes, we definitely indicate number of completed, failures, and skipped. Currently don't do time elapsed, but it's something we can think about for the future. Let me check with one our UX experts on what the best icon would be. I'm slowing being sold on your viewpoint, but I'd like to double check with them too. Thanks!

**Comment by MRayermannMSFT** [comment]
Hey @DonFrazier , finished talking to the UX folks and here's what we'll be doing.

First, the icon an activity will use should correspond to the most severe outcome for the files that are transferred. Second, we're going to change to using this icon for when a skip is the worst thing that happened:

![image](https://user-images.githubusercontent.com/21112697/60623923-d2e00f80-9d98-11e9-995b-c2f0faf5883b.png)

The error icon, as you say, is meant for errors. A skip isn't really an error. At the same time, we don't want to use the green check because for some users a skip may not be expected. I understand for your case it is, but that isn't always true. We also don't want to use the blue info icon (which I know we do for JS uploads and donwloads) as it doesn't convey the level of potential wrongness that might be occurring.

I know this isn't the outcome you were suggesting, but I hope it is at least more palatable than the current situation. Thanks!

**Comment by DonFrazier** [comment]
Seems like there could be a setting for skip/replace but instead the tool hard-codes skip.  Then a green check would tell me that existing files were skipped and everything else / nothing was downloaded without errors.  Maybe that's a different change request.

Since the tool was kind enough to give me a ready-made command that showed me how to call azcopy directly I've actually switched to it for my daily refresh needs.  Maybe they'll fix that crazy `azcopy login` SNAFU where I have to launch a browser, copy / paste a token before it works. But that seems to be only needed whenever my creds change or I get a new version.  And then, I know that's outside the scope of this feedback.

**Comment by MRayermannMSFT** [comment]
Icon change was merged into master for 1.10.0
